lib/morandi/image_ops.rb

```diff
@@ -315,4 +315,56 @@ def call(_image, pixbuf)
       end
     end
   end
+
+  class ShrinkToFit < ImageOp
+    attr_accessor :crop, :size, :print_size, :shrink
+
+    def call(_image, pixbuf)
+      return unless shrink
+
+      original_img_height = pixbuf.height # Original size
+      original_img_width = pixbuf.width # Original size
+
+      shrink_x, shrink_y, _width, _height = crop
+      output_width, output_height = print_size
+
+      surface = Cairo::ImageSurface.new(:rgb24, output_width, output_height)
+      cr = Cairo::Context.new(surface)
+
+      # Apply White background
+      cr.save do
+        cr.set_operator :source
+        cr.set_source_rgb 1, 1, 1
+        cr.paint
+        cr.fill
+      end
+      # end
+
+      cr.translate(shrink_x.abs, shrink_y.abs)
+      scale_by = largest_shrink_ratio(output_width, output_height, original_img_width, original_img_height)
+      cr.scale(scale_by, scale_by)
+      cr.set_source_pixbuf(pixbuf)
+
+      cr.paint(1.0)
+      final_pb = surface.to_pixbuf
+
+      # Clean up
+      cr.destroy
+      surface.destroy
+      final_pb
```

Comment:
```suggestion

      final_pb
```

Really picky, but it makes it clearer it's not part of the clean-up, because [Rubocop doesn't like explicit `return` statements](https://github.com/rubocop/ruby-style-guide#no-explicit-return).